### DISCUSSION TITLE
Deprecation of VLC feature

### DISCUSSION THREAD

**Discussion by skimark1** [sircharlo/meeting-media-manager#2709] [Q&A] [answered]
I wondered what the reasoning is behind discontinuing VLC list mode? I think it is a great feature that you may want to rethink the idea of dropping, especially since it is already part of the software? It really speeds things up for those of us who use VLC on a Mac at the Kingdom Hall. Thanks for your hard work on this great tool.

**Comment by e-Feitosa** [comment, marked as answer]
**The feature of adding media present in JWL made this feature of the M3 fall into disuse, that is, you can run a playlist in JWL and add media files (audio / image / video) to play directly in JWL dispensing the use of VLC for external media.**

**Comment by skimark1** [comment]
The reason we still use VLC is that the M series Macs running JWL don't have the option to share media to a second display for some reason. I've called the Branch support number to ask about it and they said they would look into it. I makes it hard to do a share to monitors in hall and to also stream it to zoom when you only have one window to work with. You can't see your play list with that constraint. Additionally, older Intel based Macs can't run JWL, so they have to rely on VLC

**Reply by sircharlo** [reply]
May I ask what the advantages of using VLC over M³ are? Perhaps there are some features that VLC implements that could be useful in M³. 
Otherwise, I find M³ to be a solid purpose-built replacement for VLC during meetings, and personally could not find a reason to favor VLC over M³, since M³ not only fetches the media but can also play it in a secondary window.

**Reply by ux-git** [reply]
@skimark1, just curious, what's a purpose of using Macs instead of the LDC-recommended Windows machines? Did your Branch support mention anything regarding this?

**Comment by skimark1** [comment]
One thing is that we have no idea how to use M3 in a KHall environment. The instruction pages on GitHub return a 404.

**Reply by sircharlo** [reply]
Sorry about that! We're in the process of creating/updating the docs with the new version.
Can you take a look [here](https://sircharlo.github.io/mmm-refactor/about) and let me know if this answers your questions?

**Reply by e-Feitosa** [reply]
@sircharlo, In the new document cites _"As for fully remote congregation Zoom meetings, the inbuilt MP4 conversion feature in M³ enables you to share media files of all types easily, using Zoom's native MP4 sharing feature.'_ this means that M3 will continue to have the option of converting images into video?

Font:
[](https://sircharlo.github.io/mmm-refactor/about#:~:text=As%20for%20fully%20remote%20congregation%20Zoom%20meetings%2C%20the%20inbuilt%20MP4%20conversion%20feature%20in%20M%C2%B3%20enables%20you%20to%20share%20media%20files%20of%20all%20types%20easily%2C%20using%20Zoom%27s%20native%20MP4%20sharing%20feature.)

**Reply by mtdvlpr** [reply]
That was copied over from the previous documentation. Might have been an oversight.

**Comment by skimark1** [comment]
We are definitely running hybrid meetings. Our problem is that JWL on M series Macs doesn’t have the option to play video to a 2nd screen enable, and I’ve contacted the Branch about this. They say they are aware of the issue and working on it. JWL on iPad has the option enabled and that is the same app that is installed on a M series Mac. That is where using VLC comes in for us, being able to see the media library during a share. Not sure if M3 does that since I don’t see any instructions and haven’t stumbled on anything in going thru M3.

Thanks,
Mark Cillis
***@***.***

**Reply by sircharlo** [reply]
Meeting Media Manager should automatically detect your second screen and use it for the media display. Is that not what's currently happening?

**Reply by sircharlo** [reply]
(Try with the [latest version](https://github.com/sircharlo/mmm-refactor/releases/latest) to be sure, as a longstanding issue preventing the media window from going fullscreen on macOS was fixed in this version.)

**Comment by skimark1** [comment]
After retrieving media, I don’t see any media controls anywhere, to be able to play directly from M3. Nothing onscreen or in menus.

Thanks,
Mark Cillis
***@***.***

**Reply by sircharlo** [reply]
Are you using the latest version? We're rolling out a brand new version in a few days, but I think from your description that you're using the legacy version. Try this one to be sure: https://github.com/sircharlo/mmm-refactor/releases/latest

**Comment by skimark1** [comment]
24.4.8 is the version that just updated. I found the instruction to use Alt+D to show controls. On a Mac that is Option+D, so that is a bit confusing since we don’t have an Alt key. Now that I have controls I’ll experiment with it to see what happens.

Thanks,
Mark Cillis
***@***.***

**Comment by skimark1** [comment]
Is there a way to pre-set the retrieved media? I have been collecting media from home and putting it in DropBox for retrieval at the Hall. Does the media have to be retrieved by the Hall computer for it to be accessible, or can it be done in advance and shared to M3?

Thanks,
Mark Cillis
***@***.***

**Reply by mtdvlpr** [reply]
Yes, there is for version 24.4.8. Maybe this documentation can help you out:

https://github.com/sircharlo/meeting-media-manager/blob/docs/locales%2Fen%2F_posts%2F2022-08-17-congregation-sync.md

**Reply by mtdvlpr** [reply]
See also the other documentation:

https://github.com/sircharlo/meeting-media-manager/tree/docs/locales%2Fen%2F_posts

**Reply by mtdvlpr** [reply]
Note that the congregation sync feature is not present in the upcoming big release (`v24.10.x`) and up.

**Comment by sircharlo** [comment]
@skimark1 Can you please try the new version I linked to? It's 24.10.5, not 24.4.8.
I'd like your feedback on that one. This is the version that will be live soon, and it **greatly** simplifies and streamlines all of the app's majoir features. Download here: https://github.com/sircharlo/mmm-refactor/releases/latest

**Comment by skimark1** [comment]
Hi Brothers,

I really like the look and feel of the new interface. I do have some observations, and maybe a bug. Including screenshots to help:

1. On launch, there is an immediate error that nothing could be retrieved for the meeting dates:
￼

2. When I imported the files I had gotten from another Mac, the results don’t stick to the numbering system which used to put them in the correct order. (See screenshots of how it looks before download and after download)
￼

3. This is how they download from DropBox folder (Notice not in correct order, even ignoring the numbering system):
￼

After manual change to ordering of files:
￼

4. Notice too that there isn’t a heading for the WT study on this view. But that might be because it didn’t automatically download anything.

5. Also, when I try to move the songs into the correct order, they can’t be dragged to the top of the screen, and then the screen automatically scroll to the place I want to inset it. It takes several moves to get the media to where it needs to land.

6. Is there a list of keyboard shortcuts that would allow moving to the next piece of media rather than manually selecting it?

7. Also, noticed that you can only have the small window or full screen rather than being able to set the screen to the size the user might need. Probably necessary when doing a hybrid ZOOM meeting.

I wasn’t able to make this version download any media for any future meeting, and that includes this Sunday and next Tuesday. Not seeing any interface that would allow manual selection of a date for media. If you use the calendar, it will return this:
￼

We won’t have any need for VLC with this new version. Just need a little tweaking possibly? I appreciate all your great work on this wonderful tool!

Thanks,
Mark Cillis
***@***.***

**Reply by mtdvlpr** [reply]
The images are not included. Try opening this discussion directly on GitHub: https://github.com/sircharlo/meeting-media-manager/discussions/2709

**Comment by skimark1** [comment]
Github isn’t allowing me to bring the picture in either. Is there another option?

It doesn't show a preview, so I don't know if what it is listing will show as a picture or not

Thanks,
Mark Cillis
***@***.***

**Comment by mtdvlpr** [comment]
![1.png](https://github.com/user-attachments/assets/6bd6b076-a505-46c0-945b-30e052ab5119)

![3.png](https://github.com/user-attachments/assets/163b8fa0-a74d-4fd1-bed0-b1b6fd03a4ba)

![5.png](https://github.com/user-attachments/assets/f1075c68-001f-460c-a2a6-10f3dd7cc375)

![2.png](https://github.com/user-attachments/assets/e76d2677-12e6-4655-93eb-b26747573bc9)

![4.png](https://github.com/user-attachments/assets/27ae80c6-0e6c-4665-90a0-f5946b18379c)

**Comment by mtdvlpr** [comment]
Continue the conversation at https://github.com/sircharlo/meeting-media-manager/issues/2773

**Comment by skimark1** [comment]
We have had/run Macs since around 2005. Our sound was run by Mac using software written by brothers, and when JWL became supported on M series Macs, we stuck with it. No issues and nothing said by LDC trainers. Several congregations in our area are running Macs.

Thanks,
Mark Cillis
***@***.***

**Comment by sircharlo** [comment]
Continuing in #2773
